docs/contracts/StMatic.md

```diff
@@ -406,233 +307,109 @@ Requirements:
-- the contract must not be paused.
-
-:::
-
-#### Parameters
-
-| Name            | Type      | Description                         |
-| --------------- | --------- | ----------------------------------- |
-| `_account`      | `address` | Address where shares will be burned |
-| `_sharesAmount` | `uint256` | Amount of shares to burn            |
-
-#### Returns
-
-Amount of totalShares after tokens burning
-
-### stop()
-
-Stop pool routine operations
-
-```sol
-function stop()
-```
-
-### resume()
+---
 
-Resume pool routine operations
-
-```sol
-function resume()
+### requestWithdraw()
+Allows withdrawal requests to be made from validators or the StMatic contract
+```solidity
+ function requestWithdraw(uint256 _amount)
 ```
-
-### setFee()
-
-Set fee rate to `_feeBasisPoints` basis points. The fees are accrued when oracles report staking results
-
-```sol
-function setFee(uint16 _feeBasisPoints)
-```
-
 #### Parameters
 
 | Name              | Type     | Description                                                    |
 | ----------------- | -------- | -------------------------------------------------------------- |
-| `_feeBasisPoints` | `uint16` | Fee expressed in basis points, 10000 BP corresponding to 100%. |
-
-### setFeeDistribution()
-
-Set fee distribution: `_treasuryFeeBasisPoints` basis points go to the treasury,
-`_insuranceFeeBasisPoints` basis points go to the insurance fund,
-`_operatorsFeeBasisPoints` basis points go to node operators.
-The sum has to be 10 000.
-
-```sol
-function setFeeDistribution(
-  uint16 _treasuryFeeBasisPoints,
-  uint16 _insuranceFeeBasisPoints,
-  uint16 _operatorsFeeBasisPoints
-)
-```
-
-#### Parameters
-
-| Name                       | Type     | Description                                                                            |
-| -------------------------- | -------- | -------------------------------------------------------------------------------------- |
-| `_treasuryFeeBasisPoints`  | `uint16` | Fee for the treasury. Expressed in basis points, 10000 BP corresponding to 100%.       |
-| `_insuranceFeeBasisPoints` | `uint16` | Fee for the insurance fund. Expressed in basis points, 10000 BP corresponding to 100%. |
-| `_operatorsFeeBasisPoints` | `uint16` | Fee for the node operators. Expressed in basis points, 10000 BP corresponding to 100%. |
-
-### setOracle()
-
-Set authorized oracle contract address to `_oracle`
-
-```sol
-function setOracle(address _oracle)
-```
-
-#### Parameters
+| `_amount` | `uint256` | Amount to withdraw. |
 
-| Name      | Type      | Description                |
-| --------- | --------- | -------------------------- |
-| `_oracle` | `address` | Address of oracle contract |
+---
 
-### setTreasury()
-
-Set treasury contract address to `_treasury`. This contract is used to accumulate the protocol treasury fee
-
-```sol
-function setTreasury(address _treasury)
+### delegate()
+Delegates the MATIC deposited into the StMatic contract equally to the validators 
```

Comment:
Delegate the deposited tokens into the StMatic contract between Lido's active operators. The tokens are delegated based on the operator's maxDelegationLimit factor.